tests/Filter/DateRangeFilterTest.php

```diff
@@ -36,10 +36,6 @@ public function testFilterEmpty(): void
             'type' => null,
             'value' => ['start' => null, 'end' => null],
         ]));
-        $filter->filter($proxyQuery, 'alias', 'field', FilterData::fromArray([
```

Comment:
why you removed this one?

Reply:
Test were failing because I changed 
```
if ($value['start'])
```
to 
```
if ($value['start'] !== null)
```

`''` is not supposed to be a supported value since the filter is used with DateTimeType which returns either a dateTime or null, isn't it ?

Reply:
We can remove it, but it could cause some BC breaks